packages/orchestration/README.md

```diff
@@ -353,19 +353,24 @@ const orchestrationClient = new OrchestrationClient({
     output_param: 'groundingOutput',
     filters: [
         {
-          id: 'filter1',
-          data_repositories: ['repository-id']
+          id: 'FILTER_ID',
+          // data_repository_type: 'vector', // optional
+          data_repositories: ['REPOSITORY_ID'],
         }
       ],
     )
 });
 
 const response = await orchestrationClient.chatCompletion({
-  inputParams: { groundingRequest: 'What is Generative AI Hub in SAP AI Core?' }
+  inputParams: { groundingRequest: 'Give me a short introduction of SAP AI Core.' }
 });
 return response.getContent();
 ```
 
+By default, the optional filter property `data_repository_type` is set to `vector`.
+Set it to `help.sap.com` to retrieve context from the SAP Help Portal.
+`REPOSITORY_ID` can also be a wildcard `*` to search all repositories.
```

Comment:
The orchestration service has [*] as default, so I would mention the case where they want to search in a specific repo.

Reply:
Oh, didn't know that this field is optional, I will adjust the doc. Thanks!!